libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -1694,7 +1692,7 @@ void ExpressionCompiler::appendExternalFunctionCall(
 	if (funKind == FunctionType::Kind::External || funKind == FunctionType::Kind::CallCode || funKind == FunctionType::Kind::DelegateCall)
 	{
 		m_context << Instruction::DUP1 << Instruction::EXTCODESIZE << Instruction::ISZERO;
-		m_context.appendConditionalInvalid();
+		m_context.appendConditionalRevert();
```

Comment:
@chriseth I'm also not completely sure about this case. Is the reasoning that the caller can't really check and prevent this scenario explicitly?